bids/modeling/statsmodels.py

```diff
@@ -659,8 +673,12 @@ def _collections_to_dfs(self, collections):
             # apply transformations
             transformations = self.node.transformations
             if transformations:
-                transformer = tm.TransformerManager(transformations['transformer'])
+                transformer = tm.TransformerManager(
+                    transformations['transformer'], keep_history=collection_history)
                 coll = transformer.transform(coll.clone(), transformations['instructions'])
+                
+                if hasattr(transformer, 'history_'):
```

Comment:
Style question: Why `history_`? If you're trying to mark it "hidden", then it would usually be `_history`.

Reply:
Ah, I hadn't encountered that convention before.

Reply:
I had but I only recently learned what it means